src/Components/Card/CardHeader.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import CheckBox from "../Buttons/CheckBox";
+import EditButton from "../Buttons/EditButton";
+import './CardStyles.css';
+import '../Buttons/EditButton.css';
+
+
+const CardHeader = (props) => {
+
+    const [isActive, setActive] = useState(true);
+
+    const handleToggle = () => {
+        setActive(!isActive);
+    };
+
+    return (
+        <div>
+            <div className={isActive ? 'card-header' : 'card-header-checked'}>
```

Comment:
For conditional classes I suggest using `classnames` package:
```suggestion
            <div className={classNames('card-header': isActive, 'card-header-checked': !isActive)}>
```
It will ease your life when you will have to cover your code by tests.